boards/xtensa/xiao_esp32s3/doc/index.rst

```diff
@@ -0,0 +1,239 @@
+.. _xiao_esp32s3:
+
+XIAO ESP32S3
+###############
```

Comment:
CI is failing. If not the image size then it might be here. Too many `#`?

Reply:
> CI is failing. If not the image size then it might be here. Too many `#`?

Although these are definitely extra `#`s, I don't see CI failing?

Reply:
When I wrote the comment before, it failed in documentation. I re-triggered and it passed.

Reply:
so... I need to delete these `#` ? @sylvioalves

Reply:
```suggestion
XIAO ESP32S3
############
```